QUEST.tsv

```diff
@@ -1741,19 +1741,19 @@ QUEST_20150401_001740	I just gave it a while ago…
 QUEST_20150401_001741	I feel dizzy. Is the medicine working? 
 QUEST_20150401_001742	Feels like the hunger went away.
 QUEST_20150401_001743	You must have a lot of questions to ask. Go ahead, Revelator. 
-QUEST_20150401_001744	Vakarine is fighting Valtruss in the Demon Prison across the portal. {nl}She won't hold it for long unless someone helps. 
-QUEST_20150401_001745	The Baron's men went in here. Although they did not make it back.{nl}I watched everything while inside the wizard. {nl}Even the screamings of Vakarine and her Cupols.
+xQUEST_20150401_001744	Vakarine is fighting Valtrus in the Demon Prison across the portal. {nl}She won't be able hold it much longer unless someone helps.
+QUEST_20150401_001745	Men under the command of the Baron have been here though they weren't able to make it back. {nl}I watched everything from inside the wizard. {nl}Even the screams of Vakarine and her Cupols.
 QUEST_20150401_001746	Now let's talk about that. {nl}Whether you are willing to save the Goddess or not.
-QUEST_20150401_001747	I hear the demons that crossed the contaminating gap and are gathered in Aushura Chapel. {nl}I wish you luck in the name of the Goddess. 
+QUEST_20150401_001747	I hear the demons that crossed the corrupting breach are gathered in Aushura Chapel. {nl}I wish you luck in the name of the Goddess. 
 QUEST_20150401_001748	Underground Mausoleum Construction Fee
 QUEST_20150401_001749	Those who kept the honor of King Zachariel and his descendants built the King's grave here. 
 QUEST_20150401_001750	[Things about the secret place is written]
-QUEST_20150401_001751	[Details of Shout the name of the King!]{nl}
```

Comment:
I think it'll be safer to keep the {nl}'s here. We don't really know where this line resides and how it displays itself yet.